types/repo/AnonymousMessageRepo/AnonymousChannelsData.ts

```diff
@@ -40,13 +41,16 @@ export interface Member {
 export interface Message {
   id: string;
   text: string;
+  other_text?: string;
   html: string;
   type: string;
   user: User;
   attachments: any[];
   latest_reactions: any[];
   own_reactions: any[];
   reaction_counts: any;
+  isSystem?: boolean;
+  textOwnMessage?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several new fields have been added to the `Message` interface. Similar to the previous comment, ensure that these new fields are properly handled wherever this interface is used. Also, consider if these fields should be optional or required based on your application's requirements.

```diff
+  other_text?: string;
+  isSystem?: boolean;
+  textOwnMessage?: string;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->